packages/channels/botpress-channel-slack/src/views/index.jsx

```diff
@@ -62,7 +62,7 @@ export default class SlackModule extends React.Component {
   }
 
   getRedictURI = () => {
-    return this.state.hostname + '/modules/botpress-slack'
+    return this.state.hostname + '/modules/channel-slack'
```

Comment:
Was this broken before? I mean we had the wrong URL which you've fixed?

Reply:
yes, because now view for `@botpress/channel-slack` is located in `https://[your-host]/modules/channel-slack`